librz/core/cconfig.c

```diff
@@ -77,6 +77,30 @@ static int compareSize(const RzAnalysisFunction *a, const RzAnalysisFunction *b)
 	return (sa > sb) - (sa < sb);
 }
 
+#define CASE_SENSITIVE   0
+#define CASE_INSENSITIVE 1
+#define CASE_SMART       2
```

Comment:
these are defined in 2 files but should probably be moved to a header file accessible to both instead.

Reply:
Thanks for your constructive comments. Apply all suggestions.